__tests__/wares/pug.test.ts

```diff
@@ -3,11 +3,9 @@ import { createServer } from 'http'
 import { InitAppAndTest } from '../../test_helpers/initAppAndTest'
 import { renderFile } from 'pug'
```

Comment:
why do you import `renderFile` here?

Reply:
i had been trying out something and forgot to remove the imports. Is there a specific reason to set
   "noUnusedLocals": false
in the tsconfig?

Reply:
i think i accidentally copied it from my another project, gonna remove the setting